R/pkg/R/DataFrame.R

```diff
@@ -2766,18 +2780,21 @@ setMethod("histogram",
             return(histStats)
           })
 
-#' Saves the content of the SparkDataFrame to an external database table via JDBC
+#' Save the content of DataFrame to an external database table via JDBC.
 #'
-#' Additional JDBC database connection properties can be set (...)
+#' Saves the content of the SparkDataFrame to an external database table via JDBC. Additional JDBC
```

Comment:
I think we have had similar discussions in another PR, but I'm not sure I follow the reasoning for L2783 to be "Save the content of" and L2785 to be "Saves the content of" - why is one singular another plural?